src/components/BlockView/BlockView.tsx

```diff
@@ -16,6 +17,9 @@ function BlockView(props: any) {
     timestamp, hash, parentHash, miner, nonce, difficulty,
     extraData, stateRoot, transactionsRoot, receiptsRoot, transactions,
   } = block;
+  const d = moment(
+    new Date(parseInt(timestamp, 16) * 1000).toISOString(),
```

Comment:
should this be a utility function we call create date object from timestamp in muiltiple places with the same formatting.